popgen/mcmcSelection.go

```diff
@@ -97,10 +119,13 @@ func GenerateCandidateThetaPrime(t Theta, data Afs, binomCache [][]float64, s Mc
 	} else {
 		likelihood = AfsLikelihood(data, alphaPrime, binomCache, s.IntegralError)
 	}
+
+	prior = priorProb(muPrime, sigmaPrime, s)
```

Comment:
One option would be to move this prior calculation up to before the t.alpha loop that calculates the likelihood.  That would allow us to not calculate the likelihood (report -inf or 0) if the prior density is -inf/0.  Maybe it is nice to have the likelihood even when the prior excludes the proposed parameters?  Maybe it would be worth the speed-up to not do the likelihood calculation?  Could even make it an option.

Reply:
in next commit